pilot/utils/utils.py

```diff
@@ -34,23 +34,38 @@ def capitalize_first_word_with_underscores(s):
     return capitalized_string
 
 
-def get_prompt(prompt_name, original_data=None):
+def get_prompt(prompt_name, model=None, original_data=None):
```

Comment:
Let's add the model after the last keyword argument (original_data) to avoid having to modify all the functions that call it.